cvl/internal/yparser/yparser.go

```diff
@@ -92,27 +95,54 @@ int lyd_data_validate_all(const char *data, const char *depData, const char *oth
 	return lyd_validate(&pData, LYD_OPT_CONFIG, ctx);
 }
 
-int lyd_multi_new_leaf(struct lyd_node *parent, const struct lys_module *module, const char *leafVal)
+struct leaf_value {
```

Comment:
This seems to be a common struct type of key:val.  Is there not one in common definition ? If not can this be added as a data type in common definition, as it is likely required in many places ?

Reply:
No, this 'C' data structure is specific to YParser wrapper layer only, for calling libyang API. It is not needed anywhere else.